### ISSUE TITLE
Jailer of Love deaggro / hate reset / ad summoning issues

### ISSUE THREAD

**Issue by perkuprightmeow** [EdenServer/community#4431]
### Checklist

- [x] I have searched for existing issues for issues like this one. The issue has not been posted. (Duplicate reports slow down development.)
- [x] I have provided reproducable steps. (No "as the title says" posts please. Provide reproducable steps even if it seems like a waste of time.)
- [ ] I have provided my client version in the details. (type /ver into your game window)

### Details

Reporting on behalf of a couple of groups that have had issues with JoL since the deaggro maintenance : 

This report is a big scattered because it's recounting multiple issues from different groups secondhand but -

JoL will deaggro mid fight ( will go white )
- After this tank could not keep hate (constant casting on PLD/RDM couldn't hold hate against BRD songs or refresh etc.)
- JoL will stop summoning ads for several minutes after deaggro / reengage (I think 4 in the case I saw first hand)

Basically the whole fight gets out of whack once the deaggro happens. You can play around this (there's been kills with no issues ; likely due to tanks standing closer / DDs on sooner, etc.)

The groups having issues aren't doing anything outside of expected strategy for the fight. I'm guessing JoL currently tracks only by sound and you're standing outside of its detection range if you're tanking it / building hate initially. 

I have video of the aftermath of the deaggro happening (weird ad timers, tank not being able to hold hate) if that's helpful at all from a developer perspective!

**Comment by 59blargedy** [comment]
resolved in July 29, 2021 update
